apps/web/src/components/AssessmentResultsTable/AssessmentResultsTable.tsx

```diff
@@ -0,0 +1,190 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { ColumnDef, createColumnHelper } from "@tanstack/react-table";
+import uniqueId from "lodash/uniqueId";
+
+import { commonMessages, getLocalizedName } from "@gc-digital-talent/i18n";
+import { AssessmentResult } from "@gc-digital-talent/graphql";
+import { notEmpty, unpackMaybes } from "@gc-digital-talent/helpers";
+import {
+  getAssessmentStepType,
+  getPoolSkillType,
+} from "@gc-digital-talent/i18n/src/messages/localizedConstants";
+
+import Table from "~/components/Table/ResponsiveTable/ResponsiveTable";
+import adminMessages from "~/messages/adminMessages";
+import {
+  AssessmentResultType,
+  AssessmentStep,
+  AssessmentStepType,
+  PoolCandidate,
+  PoolSkill,
+} from "~/api/generated";
+
+import cells from "../Table/cells";
+import { buildColumn, columnHeader, columnStatus } from "./utils";
+import { AssessmentStepResult } from "./types";
+
+const columnHelper = createColumnHelper<AssessmentStepResult>();
+
+interface AssessmentResultsTableProps {
+  poolCandidate: PoolCandidate;
+}
+
+const AssessmentResultsTable = ({
+  poolCandidate,
+}: AssessmentResultsTableProps) => {
+  const intl = useIntl();
+
+  // Get assessment steps from pool
+  const assessmentSteps: Array<AssessmentStep> = unpackMaybes(
+    poolCandidate?.pool?.assessmentSteps,
+  );
+
+  // Get all pool skills from assessment steps and remove duplicates
+  const poolSkills: Array<PoolSkill> = unpackMaybes(
+    poolCandidate?.pool?.poolSkills,
+  );
```

Comment:
Nice catch! I thought I added that in 😕 This should solve a lot of problems. I switched to testing on Storybook since it was much easier to control the data.